content/en/observability_pipelines/sources/datadog_agent.md

```diff
@@ -27,20 +27,88 @@ Use Observability Pipelines' Datadog Agent source to receive logs from the Datad
 
-{{< tabs >}}
-{{% tab "Agent configuration file" %}}
+{{% collapse-content title="Agent configuration file" level="h4" expanded=false id="id-for-anchoring" %}}
 
 {{% observability_pipelines/log_source_configuration/datadog_agent %}}
 
-{{% /tab %}}
-{{% tab "Agent Helm values file" %}}
+{{% /collapse-content %}}
+
+{{% collapse-content title="Agent Helm value file" level="h4" expanded=false id="id-for-anchoring" %}}
 
 {{% observability_pipelines/log_source_configuration/datadog_agent_kubernetes %}}
 
+{{% /collapse-content %}}
+
+[1]: /containers/docker/log/?tab=containerinstallation#linux
+[2]: /containers/guide/container-discovery-management/?tab=helm#setting-environment-variables
+
+{{% /tab %}}
+
+{{% tab "Metrics" %}}
+
+Use the Agent configuration file or the Agent Helm chart values file to connect the Datadog Agent to the Observability Pipelines Worker.
+
+**Note**: If your Agent is running in a Docker container, you must exclude Observability Pipelines metrics, such as utilization and events in/out metrics, using the `DD_CONTAINER_EXCLUDE_METRICS` environment variable. For Helm, use `datadog.containerExcludeMetrics`. This prevents duplicate metrics, as the Worker also sends its own metrics directly to Datadog. See [Docker Metrics Collection][3] or [Setting environment variables for Helm][2] for more information.
+
+{{% collapse-content title="Agent configuration file" level="h4" expanded=false id="id-for-anchoring" %}}
+
+To send Datadog Agent metrics to the Observability Pipelines Worker, update your [Agent configuration file][1] with the following:
+
+```
+observability_pipelines_worker:
+  metrics:
+    enabled: true
+    url: "http://<OPW_HOST>:8383"
+
+```
+
+`<OPW_HOST>` is the host IP address or the load balancer URL associated with the Observability Pipelines Worker.
+- For CloudFormation installs, use the `LoadBalancerDNS` CloudFormation output for the URL.
+- For Kubernetes installs, you can use the internal DNS record of the Observability Pipelines Worker service. For example: `http://opw-observability-pipelines-worker.default.svc.cluster.local:<PORT>`.
+
+**Note**: If the Worker is listening for logs on port 8282, you must use another port for metrics, such as 8383.
+
+After you restart the Agent, your observability data should be going to the Worker, processed by the pipeline, and delivered to Datadog.
+
+[1]: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Agent Helm values file" level="h4" expanded=false id="id-for-anchoring" %}}
+
+To send Datadog Agent metrics to the Observability Pipelines Worker, update your Datadog Helm chart [datadog-values.yaml][1] with the following environment variables. See [Agent Environment Variables][2] for more information.
+
+```
+datadog:
+  env:
+    - name: DD_OBSERVABILITY_PIPELINES_WORKER_METRICS_ENABLED
+      value: true
```

Comment:
ignore if not valid but going with the pattern of line 94
```suggestion
      value: "true"
```

Reply:
I don't think this one should quotes based on the other docs, so leaving it as is.